### PR TITLE
Generate ext/phar class synopses from stubs

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/phar/Phar.xml [hunk 1]
```diff
@@ -25,39 +21,58 @@
...
      <xi:fallback />
     </xi:include>
 
     <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.phar')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
- 
-<!--    <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.recursivedirectoryiterator')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[1])" />
--->
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.phar')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[not(@role='procedural')])">
```
Comment (kocsismate) [inline reference/phar/Phar.xml:53]: the include for the constructor was missing from here, even though the method is already documented

File: reference/phar/PharData/destruct.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  <refname>PharData::__destruct</refname>
+  <refpurpose>Destructs a non-executable tar or zip archive object</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <methodname>PharData::__destruct</methodname>
+   <void/>
+  </methodsynopsis>
```
Comment (cmb69) [inline reference/phar/PharData/destruct.xml:13]:
```suggestion
  <destructorsynopsis>
   <modifier>public</modifier> <methodname>PharData::__destruct</methodname>
   <void/>
  </destructorsynopsis>
```

File: reference/phar/Phar/destruct.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <refname>Phar::__destruct</refname>
+  <refpurpose>Destructs a Phar archive object</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <methodname>Phar::__destruct</methodname>
+   <void/>
+  </methodsynopsis>
```
Comment (cmb69) [inline reference/phar/Phar/destruct.xml:13]:
```suggestion
  <destructorsynopsis>
   <modifier>public</modifier> <methodname>Phar::__destruct</methodname>
   <void/>
  </destructorsynopsis>
```

File: reference/phar/PharFileInfo/destruct.xml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  <refname>PharFileInfo::__destruct</refname>
+  <refpurpose>Destructs a Phar entry object</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <methodname>PharFileInfo::__destruct</methodname>
+   <void />
+  </methodsynopsis>
```
Comment (cmb69) [inline reference/phar/PharFileInfo/destruct.xml:13]:
```suggestion
  <destructorsynopsis>
   <modifier>public</modifier> <methodname>PharFileInfo::__destruct</methodname>
   <void />
  </destructorsynopsis>
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Looks good except for the destructorsynopses.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**